InstagramKit.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.license      = 'MIT'
   s.author       = { "Shyam Bhat" => "shyambhat@me.com" }
   s.platform     = :ios, '7.0'
-  s.source       = { :git => "https://github.com/shyambhat/InstagramKit.git", :tag => s.version }
+  s.source       = { :git => "https://github.com/derek-chim/InstagramKit", :tag => s.version }
   s.source_files  = 'InstagramKit', 'InstagramKit/**/*.{h,m}'
```

Comment:
Please ignore the podspec changes, just for my temporary use only.
And have a look on commit "Add NS_SWIFT_NOTHROW for disabling error throw handling in swift	"(3252d52)